test/views/declaration/DucrChoiceViewSpec.scala

```diff
@@ -60,6 +61,15 @@ class DucrChoiceViewSpec extends PageWithButtonsSpec with Injector {
       }
     }
 
+    onEveryDeclarationJourney(withStatus(DeclarationStatus.AMENDMENT_DRAFT)) { implicit request =>
+      val view = createView()
+
+      "hide 'Back' button to the /declarant-details page" in {
```

Comment:
Need to specify that this is testing the situation where the draft dec is an amendment draft